localstripe/resources.py

```diff
@@ -291,6 +291,78 @@ def _export(self, expand=None):
 extra_apis.append(('GET', '/v1/balance', Balance._api_retrieve))
 
 
+class BalanceTransaction(StripeObject):
+    object = 'balance_transaction'
+    _id_prefix = 'txn_'
+
+    def __init__(self, amount=None, currency=None, description=None,
+                 exchange_rate=None, reporting_category=None, source=None,
+                 type=None,
+                 **kwargs):
+        if kwargs:
+            raise UserError(400, 'Unexpected ' + ', '.join(kwargs.keys()))
+
+        try:
+            assert _type(amount) is int
+            assert _type(currency) is str and currency
+            assert description is None or _type(description) is str
+            assert exchange_rate is None or _type(exchange_rate) is float
+            assert reporting_category in ('charge', 'refund')
+            assert _type(source) is str
+            assert type in ('charge', 'refund')
+        except AssertionError:
+            raise UserError(400, 'Bad request')
+
+        if source.startswith('ch_'):
+            Charge._api_retrieve(source)
+        elif source.startswith('re_'):
+            Refund._api_retrieve(source)
+        else:
+            raise UserError(400, 'Bad request')
```

Comment:
Can you add these comments? Otherwise it's not obvious for new developers why these lines are for.
```suggestion
        if source.startswith('ch_'):
            Charge._api_retrieve(source)  # to return 404 if not existant
        elif source.startswith('re_'):
            Refund._api_retrieve(source)  # to return 404 if not existant
        else:
            raise UserError(400, 'Bad request')
```